features/polyExplorer/src/screens/stories/messengerStory.css

```diff
@@ -1,29 +1,3 @@
-.messenger-parts {
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-start;
-  align-items: center;
-}
-
-.messenger-parts .one::first-letter {
-  float: left;
-  font-size: 60px;
-  line-height: 40px;
-  padding: 3px;
-}
-
-.title-messenger-story {
-  padding-left: 20px;
-  width: 85%;
-  font-weight: bold;
-  font-size: 45px;
-  line-height: 100%;
-  letter-spacing: -0.01em;
-  color: var(--color-primary-messenger-story) !important;
-}
-
-@media(min-width: 321px) {
-  .title-messenger-story {
-    width: 100%;
-  }
+.messenger .cluster-story-title {
+  color: var(--color-primary-messenger-story);
```

Comment:
I like this, for instance. "Primary color" is much more informative than "dark" or "light"